packages/legend-studio-manual-tests/src/__tests__/roundtrip-grammar/cases/basic-M2M.pure

```diff
@@ -39,7 +39,7 @@ Mapping model::mapping
   *model::target::_Firm: Pure
   {
     ~src model::Firm
-    employees[model_target__Person, x]: $src.employees,
+    employees[model_target__Person]: $src.employees,
```

Comment:
why lose the target id in the test.

Reply:
it will fail when we do rountrip

Reply:
bit confused why is the x swallowed?

Reply:
🤷

Reply:
can you explain :)

Reply:
just as confused as you are, but pretty sure we need to check how `engine` swallow `x`, and yeah, why do we have `x` there in the first place?